opentrons-ai-client/src/molecules/InputPrompt/__tests__/InputPrompt.test.tsx

```diff
@@ -55,15 +55,16 @@ describe('InputPrompt', () => {
     render()
     screen.getByRole('textbox')
     screen.queryByPlaceholderText('Type your prompt...')
-    screen.getByRole('button')
-    expect(screen.getByRole('button')).toBeDisabled()
+    const sendButton = screen.getByRole('button', { name: 'Send' })
+    expect(sendButton).toBeDisabled()
```

Comment:
nit

basically we define button const when we will need to check its existence + sytle/click action.

```suggestion
    expect(screen.getByRole('button', { name: 'Send' })).toBeDisabled()
```